syft/__init__.py

```diff
@@ -1 +1,2 @@
 from .tensor import TensorBase
+from .tensor import *
```

Comment:
@TheOriginalAlex @erfannoury what do you think of using __all__ in combination with `from .tensor import *`? I see many libraries doing this.